src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.html

```diff
@@ -214,6 +215,23 @@
           </div>
         </div>
 
+        <!-- Force change password -->
+        <div class="form-group row"
+             *ngIf="!isCurrentUser() && !authStorageService.isSSO()">
+          <div class="offset-sm-3 col-sm-9">
+            <div class="custom-control custom-checkbox">
+              <input type="checkbox"
+                     class="custom-control-input"
+                     id="pwdUpdateRequired"
+                     name="pwdUpdateRequired"
+                     formControlName="pwdUpdateRequired">
+              <label class="custom-control-label"
+                     for="pwdUpdateRequired"
+                     i18n>User must change password at next logon</label>
```

Comment:
The user should be informed that an password has to be set in order for the new user to log in. (Or you should be able to click on login without a password)

Reply:
As discussed we should do that in a follow-up PR.

Reply:
I've created the issue -> https://tracker.ceph.com/issues/43930